merlin/systems/dag/ops/feast.py

```diff
@@ -298,6 +289,7 @@ def transform(
             Transformed tensor dictionary
         """
         entity_ids = transformable[self.entity_column]
+        array_lib = entity_ids._array_lib
```

Comment:
This sets us up to return arrays from the Feast op that match whatever was passed in; if you supply the entity ids as a GPU array, you'll get GPU arrays back.